### PR TITLE
Mass account creation   traduora (updated)

### PR DESCRIPTION
**Pull request by Anon-Artist**
# Description
Massive account creation chained with Session fixation in traduora.
# Proof of Concept 
1. setup traduora to reproduce the vulnerability 
2. go to create an account page http://localhost:8080/signup
3. Append a Name, Email Address and Password and capture the request with Burpsuite 
4. Send the request to intruder and provide the necessary option and attack
For testing I used battering ram mode and created admin1 to admin50 with email admin1@hunt.com to admin50@hunt.com
* https://drive.google.com/file/d/1QzjQmYBBOSUB4G5pEtI34tqQEd2meKlo/view?usp=sharing
Traduora is also vulnerable to session fixation issue due to that once attacker registerd the accounts after that victim opens his account with new password the seesion of the attacker will not expired.
# Impact
Able to Create multiple accounts with a single request to annoy legit users that user with this email already exist and attacker is able to maintain access with old sessions even if the security of the account is updated by the victim.

### CODE REVIEW

**Comment by Anon-Artist** [comment]
@JamieSlome i had created a new branch as you suggested.
I think every conflicts were solved now
Thankyou

File: bounties/other/traduora/2/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "PackPackageVulnerabilityID":"2",
```
Comment (JamieSlome) [inline bounties/other/traduora/2/vulnerability.json:2]: Spelling mistake here!

**Review by JamieSlome — CHANGES_REQUESTED** [review]
Please refer to the requested changes - cheers! 🍰

File: bounties/other/traduora/2/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "Massive account creation",
```
Comment (JamieSlome) [inline bounties/other/traduora/2/vulnerability.json:5]: Can you change to `Mass Account Creation`?

File: bounties/other/traduora/2/vulnerability.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "Package": {
+        "Registry": "other",
+        "Name": "traduora",
+        "URL": "https://github.com/traduora/traduora",
+        "Downloads": ""
+    },
+    "CWEs": [
+        {
+            "ID": "284",
+            "Description": "Massive account creation is chained with session fixation"
```
Comment (JamieSlome) [inline bounties/other/traduora/2/vulnerability.json:19]: Change to `Mass Account Creation` 🍰

**Review by Anon-Artist — COMMENTED** [review]
Updated @JamieSlome

**Review by Anon-Artist — COMMENTED** [review]
Done :+1:

**Pull request merged**